controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -1415,6 +1425,9 @@ fields:
   - name: K8S_ClusterName
     type: string
     description: Cluster name or identifier
+  - name: EncryptedFlow
+    type: boolean
```

Comment:
Do you think we will be able to get more info that that in the future ? 

It would be interesting to use another type here if so

Reply:
in fact, I think we could simplify that from the user perspective, with just a single field "IPsec" that could be "true" / "false" / "error: <code>", wdyt?
can be a follow-up